services-turf/src/main/java/com/mapbox/turf/TurfMeasurement.java

```diff
@@ -337,6 +340,57 @@ public static Point along(@NonNull LineString line, @FloatRange(from = 0) double
     return bboxCalculator(resultCoords);
   }
 
+  /**
+   * Takes a set of features, calculates the bbox of all input features, and returns a bounding box.
+   *
+   * @param geoJson a {@link GeoJson} object
+   * @return a double array defining the bounding box in this order {@code [minX, minY, maxX, maxY]}
+   * @since 4.8.0
+   */
+  public static double[] bbox(GeoJson geoJson) {
+    BoundingBox boundingBox = geoJson.bbox();
```

Comment:
One thing. I am not convinced that we can trust the bbox that came with the geoJson.

Reply:
capturing from the spec, this can be provided though seems optional. If bbox isn't implemented, it will try to base the bbox on the actual geometries.